Header header

string[] objects_name
geometry_msgs/Point object_position
geometry_msgs/Quaternion object_orientation
geometry_msgs/Vector3 object_scale
string[] arm_name
geometry_msgs/Point goal_position
geometry_msgs/Quaternion goal_orientation

